app-office/ganttproject-bin/ganttproject-bin-2.8.5-r2179.ebuild

```diff
@@ -17,17 +17,17 @@ DEPEND="app-arch/unzip
 	>=virtual/jdk-1.4"
```

Comment:
I that this code is compatible with java 4 anymore, can you check and update the libraries?